### PR TITLE
Make Dev UI 2 the default Dev UI

### PR DESCRIPTION
**Pull request by cescoffier**
The old dev ui is now accessible from /q/dev-v1.
The new one is now accessible from /q/dev-ui or /q/dev (redirection to /q/dev-ui).

Superseedes #31973

### CODE REVIEW

File: docs/src/main/asciidoc/resteasy-reactive-migration.adoc [hunk 1]
```diff
@@ -100,7 +100,7 @@ Users are instead encouraged to use the Jakarta REST standard `jakarta.ws.rs.Bea
 
 Although RESTEasy Reactive provides the same spec compliant behavior as RESTEasy Classic does, it does not include the same exact provider implementations at runtime.
 
-The most common case where the difference in providers might result in different behavior, is the included `jakarta.ws.rs.ext.ExceptionMapper` implementations. To see what classes are included in the application, launch the application in dev mode and navigate to http://localhost:8080/q/dev/io.quarkus.quarkus-resteasy-reactive/exception-mappers.
+The most common case where the difference in providers might result in different behavior, is the included `jakarta.ws.rs.ext.ExceptionMapper` implementations. To see what classes are included in the application, launch the application in dev mode and navigate to http://localhost:8080/q/dev/resteasy-reactive/exception-mappers
```
Comment (phillip-kruger) [inline docs/src/main/asciidoc/resteasy-reactive-migration.adoc:103]: This url is now `http://localhost:8080/q/dev-ui/io,quarkus.quarkus-resteasy-reactive/exception-mappers`

**Review by phillip-kruger — CHANGES_REQUESTED** [review]
I did not run this locally, will do that tomorrow morning. For now just the one comment.  thanks @cescoffier !

**Comment by phillip-kruger** [comment]
@cescoffier CI failures seems related

File: docs/src/main/asciidoc/resteasy-reactive-migration.adoc (hunk 1 above)
Comment (cescoffier) [reply docs/src/main/asciidoc/resteasy-reactive-migration.adoc:103]: I was actually going to ask as I didn't know if it was implemented

**Comment by gsmet** [comment]
This won't be in CR1 but we need it for CR2. @cescoffier can you make sure we have a clean build before next Tuesday? Thanks!

**Review by gsmet — APPROVED** [review]
I tested it and it seems to work as advertised.

I will merge it so that it gets in CR2.

Note that we will have to update the documentation when we add new features and also probably rewrite the `dev-ui.adoc` guide to be about V2.

**Pull request merged**